testing/falkon/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Contributor: Leo <thinkabit.ukim@gmail.com>
+# Maintainer: Leo <thinkabit.ukim@gmail.com>
+pkgname=falkon
+pkgver=3.1.0
+pkgrel=0
+pkgdesc="Cross-platform Qt Web Browser"
+url="https://github.com/KDE/falkon"
+arch="all"
+license="GPL-3.0-or-later"
+depends="qt5-qtbase-sqlite"
+makedepends="cmake python3 qt5-qtbase-dev extra-cmake-modules qt5-qtdeclarative-dev
+ qt5-qtwebengine-dev xcb-util-dev libxcb-dev qt5-qtx11extras-dev qt5-qttools-dev
+ libgnome-keyring-dev"
```

Comment:
Falkon has several (optional) KDE Frameworks dependencies allowing for better integration with the Plasma desktop. Now KDE Frameworks is packaged, it would be nice to add them.

Reply:
i cp'd them from your PR